public/index.html

```diff
@@ -40,6 +40,7 @@
     You need to enable JavaScript to run this app.
   </noscript>
   <div id="root"></div>
+  <div id="outdated"></div>
```

Comment:
Note that `cypress/component-index.html` looks a lot like this file so that component tests can have the same basic index to work with. I would love to see a test for this if possible.